pkgs/stdenv/generic/check-meta.nix

```diff
@@ -328,6 +392,13 @@ let
     unfree = bool;
     unsupported = bool;
     insecure = bool;
+    # This is checked in more detail further down
+    problems = attrsOf (attrsOf any);
+    problemTypes = {
+      kind = enum problemKindsManual;
+      message = str;
+      urls = listOf str;
+    };
```

Comment:
This should be moved, this here would be for adding `meta.problemTypes =` support.